src/main/java/felix/utils/TaskList.java

```diff
@@ -0,0 +1,84 @@
+package felix.utils;
+
+import felix.task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for TaskList class.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Adds task from taskList.
+     */
+    public void addTask(Task t) {
+        this.tasks.add(t);
+    }
+
+    /**
+     * Returns task at index i from taskList.
+     */
+    public Task getTask(int i) {
+        return this.tasks.get(i);
+    }
+
+    /**
+     * Returns number of tasks in taskList.
+     */
+    public int getCount() {
+        return this.tasks.size();
+    }
+
+    /**
+     * Deletes task at index i from taskList.
+     */
+    public void deleteTask(int i) {
+        this.tasks.remove(i);
+    }
+
+    /**
+     * Returns the String representation of all tasks within taskList to be written to a file.
+     */
+    public String getFileRepresentation() {
+        StringBuilder output = new StringBuilder();
+        for (int i = 0; i < this.tasks.size(); i++) {
+            output.append(this.tasks.get(i).toFileString());
+            if (i < this.tasks.size() - 1) output.append("\n");
```

Comment:
Single statement conditionals should still be wrapped by curly brackets. I noticed the same issue in other places as well.

Reply:
Thank you for the feedback! I will fix this.